### PR TITLE
refactor component library

### LINKED ISSUE #24: Remove inline styling and pick a Styling Framework
We're using Chakra UI - switch over existing code to remove inline styling.
* https://chakra-ui.com/ 

Terms of Acceptance: 
* Chakra imported
* And inline styling removed

### PR DESCRIPTION
**Pull request by kayleewuhu**
ℹ️ Issue
Closes #24 

📝 Description
Briefly list the changes made to the code:

Refactor component library from react-bootstrap to chakra-ui
Improve UI formatting and consistency across components

✔️ Testing
Run the website and try with different screen sizes

🏕️ (Optional) Future Work / Notes
Components move around in the timesheet component, especially when the submit card changes 
Also, pages are not responsive
Additionally, log out page is not incredibly user friendly
homePage

timesheet

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -1,44 +1,67 @@
...
+                <MenuItem as='a' href={dropDownItem.link} key={dropDownItem.title}>
+                  {dropDownItem.title}
+                </MenuItem>
+              )
+            )}
+          </MenuList>
+        </Menu>
+        <Spacer />
+        <Button as='a' href={PAGE_ROUTES.LOGOUT} rounded={'md'} bgColor={defaultColors.BREAKTIME_BLUE} textColor={'white'}>
+          Sign out
```
Comment (SofieDunt) [inline src/components/NavBar/NavBar.tsx:60]: maybe
```suggestion
          Sign Out
```

**Review by SofieDunt — COMMENTED** [review]
Nice! Can you post screenshots in the PR description?

**Review by SofieDunt — COMMENTED** [review]
Nice! Can you post screenshots in the PR description?

File: src/components/SignOut/Signout.tsx [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
   return (
-    <div className="App"> 
-      <header className="App-header">
-        <h3>Logged out</h3>
-        
-      </header>
-    </div>
-  ); 
+    <Heading>
+      Logged out
```
Comment (circlegov) [inline src/components/SignOut/Signout.tsx:10]: To match Sign Out maybe?
```suggestion
      Logged Out
```

**Review by izzyconner — APPROVED** [review]
lgtm - just make sure to resolve the merge conflicts first

**Pull request merged**